player/tests/test.rs

```diff
@@ -20,6 +20,7 @@ use std::{
     path::{Path, PathBuf},
     ptr, slice,
 };
+use wgt::BufferSize;
```

Comment:
let's just spell it out explicitly like we do for the other types in `wgt`, no need to include